commands/version/README.md

```diff
@@ -161,19 +161,24 @@ lerna version --git-remote upstream
 
 When run with this flag, `lerna version` will push the git changes to the specified remote instead of `origin`.
 
-### `--github-release`
+### `--release`
 
 ```sh
-lerna version --github-release --conventional-commits
+lerna version --release --conventional-commits
```

Comment:
Let's reverse the order of the flags and show both option values, diffing the end-of-line is easier to read
```suggestion
lerna version --conventional-commits --create-release github
lerna version --conventional-commits --create-release gitlab
```